entrypoint.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+set -e
+
+# Create the user with ssh access
+useradd -d "/home/${SSH_USER}" -m -s "/bin/bash" "${SSH_USER}"
+mkdir -p /home/${SSH_USER}/.ssh/
+
+# Configure ssh_config
+mkdir -p /var/run/sshd
+if ! grep "PasswordAuthentication no" /etc/ssh/ssh_config ; then
```

Comment:
@JesusZapata This should be /etc/ssh/ssh**d**_config (the configuration for the daemon)